<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HLAWindow</class>
 <widget class="QWidget" name="HLAWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Connections</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Available Connections</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>19</number>
      </property>
      <item>
       <widget class="QListWidget" name="mNetworkListWidget">
        <property name="toolTip">
         <string>Select an existing network connection from the list. </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="mNewPushButton">
          <property name="toolTip">
           <string>Create a new network connection. </string>
          </property>
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mEditPushButton">
          <property name="toolTip">
           <string>Edit an existing network connection. </string>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mDeletePushButton">
          <property name="toolTip">
           <string>Delete the currently selected network connection. </string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Current Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>-1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="mCurrentConnectionLineEdit">
          <property name="toolTip">
           <string>The current federation you are connected to.</string>
          </property>
          <property name="text">
           <string>None</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mConnectPushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Connect to the currently selected federation. </string>
          </property>
          <property name="text">
           <string>Connection Button</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="mReconnectOnStartupCheckBox">
          <property name="toolTip">
           <string>Check this box if you want the Stealth Viewer to reconnect to the previous connection on startup.</string>
          </property>
          <property name="text">
           <string>Reconnect On Startup</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="mAutoReconnect">
          <property name="text">
           <string>Auto Reconnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mSecondsLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Every</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mAutoReconnectDelayTime">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mEveryLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Seconds</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mClosePushButton">
         <property name="toolTip">
          <string>Close this window. </string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mNewPushButton</tabstop>
  <tabstop>mEditPushButton</tabstop>
  <tabstop>mDeletePushButton</tabstop>
  <tabstop>mClosePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mClosePushButton</sender>
   <signal>clicked()</signal>
   <receiver>HLAWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mAutoReconnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>mAutoReconnectDelayTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mAutoReconnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>mEveryLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mAutoReconnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>mSecondsLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
